postgres_exporter/linux/crontab.txt

```diff
@@ -3,6 +3,10 @@
 # Copyright © 2017-2023 Crunchy Data Solutions, Inc. All Rights Reserved.
 #
 ###
-# The below script must be run as a database superuser. You may have to adjust your pg_hba.conf to allow this and/or use a .pgpass file
+# The below script must be run as a user with the pg_read_all_data role or a superuser. You may have to adjust your pg_hba.conf to allow this and/or use a .pgpass file
 # By default only scans and stores statistics for objects that are at least 1GB in size. Adjust as needed. Set --quiet twice to suppress all output.
-# 08 03 * * 0 /usr/bin/pg_bloat_check.py --quiet --quiet -c "dbname=postgres user=postgres" --min_size=1073741824
+# 08 03 * * 0 /usr/bin/pg_bloat_check.py --quiet --quiet -c "dbname=postgres user=ccp_monitoring" --min_size=1073741824
+
+# If using the materialized views for certain metrics, set the below cronjob to run at least as often as the lowest configured interval
```

Comment:
> lowest configured interval
is that scrape interval? or some other interval?

Reply:
The interval in the configuration table below

Reply:
Updated to say "refresh interval"